content/concepts/terminology.md

```diff
@@ -3,9 +3,17 @@ title: Terminology
 description: Terminology specific to Ocean Protocol.
 ---
 
+## Ocean Network
+
+Any EVM-compatible network where all<super>\*</super> the Ocean Protocol smart contracts ([keeper contracts](https://github.com/oceanprotocol/keeper-contracts)) are deployed. There can be many Ocean networks. Examples include the [testnets](/concepts/testnets/) and [the Ocean Production Network](/concepts/production-network/).
```

Comment:
you could try using the extended markdown syntax for footnotes, so they also behave like footnotes (linked to footnote, footnote has link to go back to reference):

```markdown
Any EVM-compatible network where all[^1] the Ocean Protocol smart contracts...

...

[^1]: The "Dispenser" smart contract should only be deployed to testnets.
```

Don't think I specifically put in footnotes styling but it should return something usable.

Reply:
I tried that out and it worked! Check it out:

![Screenshot from 2019-06-12 14-42-08](https://user-images.githubusercontent.com/1210951/59352243-78670c80-8d20-11e9-9742-4a69c615eca3.png)

and then clicking on the little superscript 1 takes my browser to this footnote near the bottom of the page:

![Screenshot from 2019-06-12 14-42-29](https://user-images.githubusercontent.com/1210951/59352246-7c932a00-8d20-11e9-8bac-1a3950444af8.png)

Reply:
nice!